proposals/0002-root-signature-in-clang.md

```diff
@@ -414,6 +414,50 @@ parsedRootSignature = RootSignature{
 };
 ```
 
+### Default Values of Optional Parameters
+
+Many of the parameters of each root element are optional. If they are not
+specified they will take the following default values. These comply with
+previous documentation
+[here](https://learn.microsoft.com/en-us/windows/win32/direct3d12/specifying-root-signatures-in-hlsl).
+
+General Parameters:
+
+- `visibility = SHADER_VISIBLITY_ALL`
+
+Descriptor Specific:
```

Comment:
Looks like this is a mixture of root descriptor and descriptor range specific ones.  Would be good to get those split out and explicit.